proto/regen/ecocredit/v1alpha1/types.proto

```diff
@@ -71,8 +71,18 @@ message BatchInfo {
 // use with the x/params module.
 message Params {
   // credit_class_fee is the fixed fee charged on creation of a new credit class
-  repeated cosmos.base.v1beta1.Coin credit_class_fee = 1
-      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  repeated cosmos.base.v1beta1.Coin credit_class_fee = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
+
+  // allowed_class_creator_addresses is an allowlist defining the addresses with
+  // the required permissions to create credit classes
+  repeated string allowed_class_creator_addresses = 2;
```

Comment:
I guess for consistency we might want to change this to `allowed_class_designer_addresses`?

Reply:
ah yeah, good catch - fixed here https://github.com/regen-network/regen-ledger/pull/426/commits/95a4f7183353835e749a7998a2ccc286500f675d